app/views/categories/index.html.erb

```diff
@@ -20,6 +20,11 @@
                 </a>
               </h3>
               <div itemprop='description'><%= c.description&.html_safe %></div>
+              <div class='subcategories'>
```

Comment:
Should we have a quick test to ensure there's at least one subcategory before adding this `div`?

Reply:
Yes, certainly, good call.